### ISSUE TITLE
[Question] Dataset Silence

### ISSUE THREAD

**Issue by queechy** [SJTMusicTeam/Muskits#136]
I'm creating a dataset using human-made songs, but should I remove the Silence of the songs?

**Comment by ftshijt** [comment]
Yeah, we do not intend to process silence. If you check our released recipes, you can find usually we would have a stage in the pre-processing (usually in `local/data.sh` for each recipe folder) that split the songs into non-silence segments based on the silence information.
